### ISSUE TITLE
Option to change keyboard shortcuts

### ISSUE THREAD

**Issue by ChocolateLoverRaj** [pop-os/cosmic-term#197]
I'm guessing that there is already an issue about this in a different repo but idk. I would like the ability to re-bind paste to Ctrl+V.

**Comment by juhp** [comment]
(Just commenting that using Ctrl+v (more so Ctrl+c) in a terminal generally not considered good for paste (and copy), since some (terminal) programs use those bindings already (for example emacs, which also uses Ctrl-x). I admit it might be a bit annoying to adapt to, I often conversing pressing Ctr+Shift+c in my browser by mistake.)

**Comment by juhp** [comment]
I would like to see configuration for Tab switching shortcuts.

eg I want to use:
- Alt+PageUp instead of Ctrl+Shift+Tab
- Alt+PageDown instead of Ctrl+Tab
- Alt+[1-9] instead of Ctrl+Shift+[1-9]

due to my long muscle memory coming from gnome-terminal and other terminals.

**Comment by rswinkle** [comment]
I too just upgraded to Pop_OS 24.04 and immediately ran into not being able to change the terminal shortcuts as a huge workflow/muscle memory issue.

I don't use tiling and my workflow is a multi-tabbed terminal window with (neo)vim in one of them so I always make the following changes which are much more ergonomic and intuitive for a vim user (especially with my custom ergodox layout letting me do both shortcuts without moving my hands at all)

Next Tab is CTRL+SHIFT+L
Prev Tab is CTRL+SHIFT+H

Will the ability to change Cosmic terminal keyboard shortcuts be coming in the near future or ever?

**Comment by GibsonSWE** [comment]
I too would like to be able to change keyboard shortcuts for V-split and H-split as I am super used to the Windows 11 Terminal shortcuts Win+Shift+[-] and Win+Shift+[+]

**Comment by SeSodesa** [comment]
Yeah, the default key combination for changing tabs in COSMIC Terminal is very unergonomic. I would also prefer <kbd>Some modifier</kbd> + <kbd>Page Up</kbd> / <kbd>Page Down</kbd>. I get it: using <kbd>Tab</kbd> to change tabs is kind of intuitive, but it does not jive well with my hands having to be moved away from the bottom row of the keyboard.

**Comment by forrestbao** [comment]
I want Alt + [1-9] too!

**Comment by borijang** [comment]
This is my biggest gripe currently on 24.04, which I guess it's a good thing. But yes, it would be great to be able to change the keybinds similarly to the gnome terminal.

**Comment by notlaast** [comment]
I've searched for an update about this topic multiple times in the past month, and I only just now discovered there's significant progress on this PR which would close this request, I'm sure this will be well received: https://github.com/pop-os/cosmic-term/pull/674

**Comment by grscheller** [comment]
#### First

It would be nice to have a place under ~/.config/cosmic/com.system76.CosmicTerm to put user maintained configuration files. I tire of having to manually configure Cosmic Term from scratch every time I set up a new workstation. 

#### Second

There is a lot of functionality in the the underlying Alacritty engine not exposed by the CosmicTerm GUI. For example, ScrollHalfPageUp is not in the [Action enum in the code](https://github.com/pop-os/cosmic-term/blob/a43b5f7d/src/main.rs#L220-L266). I can understand not making the GUI too overwhelming, but having more actions available to assign keybindings to in a user maintain config file would be nice.

**Comment by grscheller** [comment]
#### Not as easy as I thought

ScrollHalfPageUp exists in Alacritty's application layer, its own keybinding and action system, not in the alacritty_terminal engine you import.

Any case, having a way to configure things via a flat, easily parsed file (TOML or YAML) would be nice.
